src/touchable.tsx

```diff
@@ -20,6 +20,7 @@ const redirect = (url: string, external = false): void => {
 
 const useStyles = createUseStyles(() => ({
     touchable: {
+        lineHeight: 'inherit',
```

Comment:
Is this causing the height difference in rows? Is this expected?

If that's the case, this change will produce *a lot* of screenshot updates in webapp

Reply:
what motivated this change?

Reply:
I need to review this because I may not need this change

Reply:
No, I think I don't need it, reverted.